modules/statistics/src/main/frontend/package.json

```
@@ -0,0 +1,56 @@
+{
+    "name": "cards-statistics",
+    "version": "0.1.0",
```

```suggestion
    "version": "0.9-SNAPSHOT",
```

But maybe we should have this as `${project.version}`...